components/admin_console/billing/billing_subscriptions/index.tsx

```diff
@@ -108,11 +109,11 @@ const BillingSubscriptions = () => {
         pageVisited('cloud_admin', 'pageview_billing_subscription');
 
         if (actionQueryParam === 'show_purchase_modal') {
-            onUpgradeMattermostCloud();
+            onUpgradeMattermostCloud('admin_console_billing_billing_subscriptions_action_query_param');
```

Comment:
Is this when it's called from the link? Lets update `action_query_param` to something more understandable for analytics. Added a suggestion but open to other options
```suggestion
            onUpgradeMattermostCloud('billing_subscriptions_external_direct_link');
```